modules/animation/include/modules/animation/animationcontroller.h

```diff
@@ -115,6 +113,7 @@ class IVW_MODULE_ANIMATION_API AnimationController : public AnimationControllerO
     DoubleMinMaxProperty playWindow;
     DoubleProperty framesPerSecond;
     TemplateOptionProperty<PlaybackMode> playMode;
+    TemplateOptionProperty<PlaybackDirection> playbackDirection;
```

Comment:
is the redundant with respect to a negative framesPerSecond? or do we always have a positive fps?

Reply:
Hmm, I wonder if the implementation of the AnimationController was finished. There are a number of unused properties, such as framesPerSecond.
Nevertheless, specifying a negative fps seems a bit too hidden to me. The most natural way would probably be to add a "play backwards" action button instead of having this dropdown.

Reply:
This in the internal representation It has little to do with the how we present it in the GUI. And the internal representation should have a little redundant information as possible. It here is redundant information then you are also responsible for making sure that that information never get out of sync. Which is often very had to do.

And yes all of the animation suff is still WIP

Reply:
Removed the duplicate state and instead use the sign of deltaTime_